<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountChooser</class>
 <widget class="QDialog" name="AccountChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="docGiaButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Độc giả</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="thuThuButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Thủ thư</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quanLyButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Quản lý</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
